shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3176,38 +3184,16 @@ v8::Local<v8::Promise> WebContents::CapturePage(gin::Arguments* args) {
     bitmap_size = gfx::ScaleToCeiledSize(view_size, scale);
 
   view->CopyFromSurface(gfx::Rect(rect.origin(), view_size), bitmap_size,
-                        base::BindOnce(&OnCapturePageDone, std::move(promise)));
+                        base::BindOnce(&OnCapturePageDone, std::move(promise),
+                                       std::move(capture_handle)));
   return handle;
 }
 
-void WebContents::IncrementCapturerCount(gin::Arguments* args) {
-  gfx::Size size;
-  bool stay_hidden = false;
-  bool stay_awake = false;
-
-  // get size arguments if they exist
-  args->GetNext(&size);
-  // get stayHidden arguments if they exist
-  args->GetNext(&stay_hidden);
-  // get stayAwake arguments if they exist
-  args->GetNext(&stay_awake);
-
-  std::ignore = web_contents()
-                    ->IncrementCapturerCount(size, stay_hidden, stay_awake)
-                    .Release();
-}
-
-void WebContents::DecrementCapturerCount(gin::Arguments* args) {
-  bool stay_hidden = false;
-  bool stay_awake = false;
-
-  // get stayHidden arguments if they exist
-  args->GetNext(&stay_hidden);
-  // get stayAwake arguments if they exist
-  args->GetNext(&stay_awake);
+// TODO(codebytere): remove in Electron v23.
+void WebContents::IncrementCapturerCount(gin::Arguments* args) {}
```

Comment:
I approve of this change but I think it needs to be `semver/major` rather than minor, since pre-existing code that uses increment() to turn on stay_hidden or stay_awake will no longer stay hidden or awake?

Reply:
restored this to keep it semver-minor.